OSDOCS3755/howto-configure-ovn-kubernetes.md

```diff
@@ -0,0 +1,137 @@
+
+<Note: To check with Oren, Jerome> If you choose to install and use the CLI locally, this tutorial requires that you're running the Azure CLI version 2.6.0 or later. Run az --version to find the version. For details on installing or upgrading Azure CLI, see [Install Azure CLI](/cli/azure/install-azure-cli).
+
+1. Use the following URL to download both the Python wheel and the CLI extension:
+
+    [https://aka.ms/az-aroext-latest.whl]
+
+2. Run the following command:
+
+```azurecli-interactive
+az extension add --upgrade -s aro-1.0.6-py2.py3-none-any.whl
+```
+
+3. Verify the CLI extension is being used:
+
+```azurecli-interactive
+az extension list
+[
+  {
+    "experimental": false,
+    "extensionType": "whl",
+    "name": "aro",
+    "path": "<path may differ depending on system>",
+    "preview": true,
+    "version": "1.0.6"
+  }
+]
+```
+
+2. Run the following command:
+
+```azurecli-interactive
+az aro create --help
+```
+
+The result should show the –sdn-type option, as follows:
+
+```json
+--sdn-type --software-defined-network-type : SDN type either "OpenShiftSDN" (default) or "OVNKubernetes". Allowed values: OVNKubernetes, OpenShiftSDN
+```
+
+## Verify your permissions
+
+Using OVN CNI network provider for Azure Red Hat OpenShift clusters requires you to create a resource group, which will contain the virtual network for the cluster. You must have either Contributor and User Access Administrator permissions, or Owner permissions, either directly on the virtual network, or on the resource group or subscription containing it.
+
+You will also need sufficient Azure Active Directory permissions (either a member user of the tenant, or a guest user assigned with role Application administrator) for the tooling to create an application and service principal on your behalf for the cluster. For more information about user roles, see [Member and guest users](../active-directory/fundamentals/users-default-permissions#member-and-guest-users) and [Assign administrator and non-administrator roles to users with Azure Active Directory](../active-directory/fundamentals/active-directory-users-assign-role-azure-portal).
+
+## Register the resource providers
+
+You must register the resource providers if you have multiple Azure subscriptions. For information about the registration procedure, see [Register the resource providers](tutorial-create-cluster#register-the-resource-providers).
+
+## Create a virtual network containing two empty subnets
+If you have an existing virtual network that meets your needs, you can skip this step. To know the procedure of creating a virtual network, see [Create a virtual network containing two empty subnets](tutorial-create-cluster#create-a-virtual-network-containing-two-empty-subnets).
+
+## Create an Azure Red Hat OpenShift cluster by using OVN-Kubernetes CNI network provider
+
+Run the following command to create an Azure Red Hat OpenShift cluster that uses the OVN CNI network provider:

```

Comment:
More so a clarification question/ consistency nit. There are some cases, like here and line `119` where you use `using the OVN CNI network provider`. But on lines `43` and `44` you use just `using OVN CNI network provider`. I personally think having the `the` is clearer, but I mainly wasn't sure if you were leaving out the `the` on purpose.